frontend/src/app/testQueue/TestCardForm/testUtils/testConstants.ts

```diff
@@ -0,0 +1,183 @@
+import { QueriedFacility, QueriedTestOrder } from "../types";
```

Comment:
Thanks for creating this file! Makes the tests less overwhelming to read through!